src/NLog/Layouts/JsonAttribute.cs

```diff
@@ -40,9 +40,6 @@ namespace NLog.Layouts
     /// JSON attribute.
     /// </summary>
     [NLogConfigurationItem]
-    [ThreadAgnostic]
-    [ThreadSafe]
-    [AppDomainFixedOutput]
```

Comment:
So this is not needed anymore for all attributes?

Reply:
No objects that are just marked with `NLogConfigurationItem` (To signal they are safe for NLog to probe) doesn't need the extra attributes. The extra attributes are only for Layout and LayoutRenderers.

Reply:
Could we please add that to the docs of those attributes? Then I will merge it 

(please no force push)

Reply:
Not sure I understand. Should I write that attributes intended for Layout and LayoutRenderers should only need to be used for Layout and LayoutRenderers ?

Reply:
Ah it was already there. Surprising!